interface/forms/newpatient/common.php

```diff
@@ -248,6 +248,29 @@ class="css_button link_submit" target='Main' onClick="top.restoreSession()">
 			</div>
 		</td>
      </tr>
+	<?php if($GLOBALS['set_pos_code_encounter']){ ?>
+        <tr>
+            <td><span class='bold' nowrap><?php xl('POS Code','e'); ?>: </span></td>
+            <td colspan="6">
+                <select name="pos_code">
+                <?php
+				
+                $pc = new POSRef();
+
+                foreach ($pc->get_pos_ref() as $pos) {
+                    echo "<option value=\"" . $pos["code"] . "\" ";
+					if($pos["code"] == $result['pos_code']) echo "selected";
+                    echo ">" . $pos['code']  . ": ". $pos['title'];
```

Comment:
Actually lets translate the title:
``````
echo ">" . text($pos['code'])  . ": ". xlt($pos['title']);